### PR TITLE
Add community bootnode address

### PR DESCRIPTION
**Pull request by MemberIT**
Add community bootnode address in Germany

### CODE REVIEW

**Comment by polleoalex** [comment]
enode://be128c3b3ac58f4ecd4beefbbe4136773c20a822e10edddb99b3769009475156dde442bc5e2dcf363047b86dd458cb6de832286bec0c4c5ccd9efc70166d4d97@70.34.216.171:21000

**Review by polleoalex — CHANGES_REQUESTED** [review]
enode://be128c3b3ac58f4ecd4beefbbe4136773c20a822e10edddb99b3769009475156dde442bc5e2dcf363047b86dd458cb6de832286bec0c4c5ccd9efc70166d4d97@70.34.216.171:21000

File: README.md [hunk 1]
```diff
@@ -213,4 +219,4 @@ cd ui
 docker-compose up ui
 ```
```
Comment (polleoalex) [inline README.md:220]: enode://be128c3b3ac58f4ecd4beefbbe4136773c20a822e10edddb99b3769009475156dde442bc5e2dcf363047b86dd458cb6de832286bec0c4c5ccd9efc70166d4d97@70.34.216.171:21000

Comment (setar) [reply README.md:220]: recheck plz nodeid after restart docker container. see https://github.com/masa-finance/masa-node-v1.0/issues/28

**Comment by teslashibe** [comment]
@restevens402 please review. lgtm

**Comment by MemberIT** [comment]
I fix connect to port 22001

**Review by teslashibe — APPROVED** [review]
Tested and connechttps://www.dropbox.com/s/l1rz38jqzyso3ie/Screen%20Shot%202022-02-26%20at%207.35.06%20PM.png?dl=0
Screen Shot 2022-02-26 at 7 38 48 PM
ts to the bootnode. You are eligible for the bootnode bounty program :)

**Comment by teslashibe** [comment]
@MemberIT can you reabse to main as we pushed a new bootnode to the readme and fix conflicts, please? 

Can you confirm the node port as I was able to connect on port 21000 tonight when I tested

**Comment by polleoalex** [comment]
> @restevens402 please review. lgtm

Hello. Could you check my boot node and if it fits, add me to the bounty program?
"enode://5a25539fdfbd8be0c42bc3070f05f3f8170840f9772fafae157c3bc849a179507b3aea14638944ee58de55133a63e1f1d9248aaa1e3a86e3cbe3caf3c40d6f96@136.244.80.248:21000"

File: README.md (hunk 1 above)
Comment (polleoalex) [reply README.md:226]: Hello. I've solved my problem. Could you check my boot node and if it fits, add me to the bounty program?
"enode://5a25539fdfbd8be0c42bc3070f05f3f8170840f9772fafae157c3bc849a179507b3aea14638944ee58de55133a63e1f1d9248aaa1e3a86e3cbe3caf3c40d6f96@136.244.80.248:21000"

**Comment by MemberIT** [comment]
@polleoalex сreate a pull request adding your node to the list of boot nodes

**Comment by polleoalex** [comment]
> @polleoalex сreate a pull request adding your node to the list of boot nodes

i did https://github.com/polleoalex/masa-node-v1.0/pull/1

**Pull request merged**